diff --git a/android/build.gradle b/android/build.gradle
index 211169aac0b3133ce4dab16ade7d7956c8949815..fa2defdd3e35531c6282f89cfd8db6cd1c262277 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -4,7 +4,7 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
+    mavenCentral()
   }
 
   dependencies {
@@ -53,7 +53,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
